CMakeLists.txt

```diff
@@ -13,8 +13,6 @@
 
 cmake_minimum_required(VERSION 3.18)
 
-cmake_policy(SET CMP0091 OLD)
-
```

Comment:
This should be set to `NEW` instead. From the CMake documentation:

> This policy was introduced in CMake version 3.15. Use the [`cmake_policy()`](https://cmake.org/cmake/help/v3.18/command/cmake_policy.html#command:cmake_policy) command to set it to `OLD` or `NEW` explicitly. Unlike many policies, CMake version 3.18.6 does not warn when this policy is not set and simply uses `OLD` behavior.

If the `NEW` behaviour causes problems for us it would be useful to catch it now instead of being surprised later.